exercises/randomgenerator.js

```diff
@@ -1,20 +1,34 @@
 
 
 class RandomGenerator{
-	constructor(){ 
+	constructor(minValue, maxValue){
+		this.minValue = minValue;
+		this.maxValue = maxValue;
+		if (minValue === undefined || maxValue === undefined){
```

Comment:
you should check if they are undefined first, then change the properties.  Or else for a short amount of time, they are undefined and that could be disastrous.